### PR TITLE
Beta features for testing cache buildup

### PR DESCRIPTION
**Pull request by joewalsh**
- Ability to load older days into the feed
- Ability to automatically view & save random articles

### CODE REVIEW

File: Wikipedia/Code/WMFDatabaseHouseKeeper.swift [hunk 1]
```diff
@@ -33,7 +33,7 @@ import Foundation
  
         */
         
-        guard let thirtyDaysAgoMidnightUTC = daysBeforeDateInUTC(days: -30, date: Date()) else {
+        guard let thirtyDaysAgoMidnightUTC = daysBeforeDateInUTC(days: 0 - WMFExploreFeedMaximumNumberOfDays, date: Date()) else {
```
Comment (amrox) [inline Wikipedia/Code/WMFDatabaseHouseKeeper.swift]: (minor) the name of the variable `thirtyDaysAgoMidnightUTC` is a little misleading now

File: Wikipedia/Code/WMFExploreViewController.m [hunk 2]
```diff
@@ -290,21 +295,27 @@ - (void)updateFeedSources:(nullable dispatch_block_t)completion {
...
-        //TODO: nearby doesnt always fire
         [group enter];
 #if DEBUG
         NSString *classString = NSStringFromClass([obj class]);
         [entered addObject:classString];
 #endif
-
-        [obj loadNewContentForce:NO
-                      completion:^{
+        dispatch_block_t completion = ^{
```
Comment (amrox) [inline Wikipedia/Code/WMFExploreViewController.m]: are you intentionally shadowing the `completion` parameter here?

File: Wikipedia/Code/WMFDatabaseHouseKeeper.swift (hunk 1 above)
Comment (joewalsh) [inline Wikipedia/Code/WMFDatabaseHouseKeeper.swift]: Updated & fixed some logic issues (the helper function wasn't using 'days' and also had an unnecessary UTC conversion)

File: Wikipedia/Code/WMFExploreViewController.m (hunk 2 above)
Comment (joewalsh) [inline Wikipedia/Code/WMFExploreViewController.m]: Nope, fixed

**Pull request merged**